ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -35,6 +35,11 @@ echo
 echo "${COLOR_YELLOW}Starting docker build $(date) with $(nproc) cores${COLOR_RESET}"
 echo
 
+echo "================================================================"
+echo "Verify formatting $(date)"
```

Comment:
While you're in this file, lines 61 and 62 (the "ctest" and the "echo Running tests") seem to be out of order.